### PR TITLE
link to Service Connect plugin

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

File: content/docs/apps/using-ssh.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Here's how to get a SSH shell in the environment where your app is running. You
...
+You can get a shell via the [`cf ssh`](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-command) command, which lets you securely login to an application instance where you can perform debugging, environment inspection, and other tasks. You'll want to [configure your session to match your application's environment](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-env).
 
-You can also interact directly with your the services bound to your application via [port-forwarding](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html). This allows you to access those services using native clients on your local machine.
+You can also interact directly with your the services bound to your application via [port-forwarding](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html). This allows you to access those services using native clients on your local machine. The [Service Connect plugin](https://github.com/18F/cf-service-connect#readme) makes this even easier.
```
Comment (jmcarp) [inline content/docs/apps/using-ssh.md]: "with your the services"

File: content/docs/apps/using-ssh.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Here's how to get a SSH shell in the environment where your app is running. You
 {{% govcloud %}}
 ### CF SSH
 
-You can get a shell via the [`cf ssh`](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-command) command, which lets you securely login to an application instance where you can perform debugging, environment inspection, and other tasks. You'll want to [configure your session to match your application's environment](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-env). 
+You can get a shell via the [`cf ssh`](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-command) command, which lets you securely login to an application instance where you can perform debugging, environment inspection, and other tasks. You'll want to [configure your session to match your application's environment](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-env).
```
Comment (jmcarp) [inline content/docs/apps/using-ssh.md]: > Log in is two words when it functions as a verb. For example, you log in with your login information.

http://grammarist.com/spelling/log-in-login/

**Comment by mogul** [comment]
Tagging @brittag here as it's related to #686... There's a weird overlap/division that's not feeling right about these two pages.

**Comment by brittag** [comment]
@mogul Agreed! I'll merge this one as an incremental improvement, and we can use https://github.com/18F/cg-site/pull/686 as a place to think about an overall refactor of these two pages.

**Review by brittag — APPROVED** [review]

**Pull request merged**